src/hotspot/share/opto/partialEscape.hpp

```diff
@@ -317,22 +353,26 @@ class PEAState {
   void print_on(outputStream* os) const;
 #endif
 
-#ifdef ASSERT
-  void validate() const;
-#endif
+protected:
+  using ObjColorTable = ResourceHashtable<ObjID, bool, 23, /*table_size*/
```

Comment:
comment for purpose of `ObjColorTable`? Maybe `ObjBitMap` makes more sense.

Reply:
using BitMap?